CRITICAL - 2023-10-20 16:24:47 --> Invalid file: "home/home.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('home/home.php')
 2 SYSTEMPATH\Common.php(1188): CodeIgniter\View\View->render('home/home', [], true)
 3 APPPATH\Controllers\Home.php(9): view('home/home')
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\app_web\\public\\index.php')
ERROR - 2023-10-20 17:12:37 --> mysqli_sql_exception: Unknown column 'idGrado' in 'field list' in C:\xampp\htdocs\app_web\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\app_web\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `Al...', 0)
#1 C:\xampp\htdocs\app_web\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `Al...')
#2 C:\xampp\htdocs\app_web\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `Al...')
#3 C:\xampp\htdocs\app_web\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `Al...', Array, false)
#4 C:\xampp\htdocs\app_web\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\app_web\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\app_web\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\app_web\app\Controllers\Alumno.php(74): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\app_web\system\CodeIgniter.php(919): App\Controllers\Alumno->insert()
#9 C:\xampp\htdocs\app_web\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno))
#10 C:\xampp\htdocs\app_web\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\app_web\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\app_web\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2023-10-20 17:12:37 --> Unknown column 'idGrado' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `Alumno` (`nombre`, `sexo`, `fechaNacimiento`, `idGrado`, `idGrupo`) VALUES (:nombre:, :sexo:, :fechaNacimiento:, :idGrado:, :idGrupo:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], false)
 5 APPPATH\Controllers\Alumno.php(74): CodeIgniter\Model->insert([...], false)
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Alumno->insert()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\app_web\\public\\index.php')
ERROR - 2023-10-20 17:15:48 --> mysqli_sql_exception: Unknown column 'idGrado' in 'field list' in C:\xampp\htdocs\app_web\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\app_web\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `Al...', 0)
#1 C:\xampp\htdocs\app_web\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `Al...')
#2 C:\xampp\htdocs\app_web\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `Al...')
#3 C:\xampp\htdocs\app_web\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `Al...', Array, false)
#4 C:\xampp\htdocs\app_web\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\app_web\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\app_web\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\app_web\app\Controllers\Alumno.php(74): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\app_web\system\CodeIgniter.php(919): App\Controllers\Alumno->insert()
#9 C:\xampp\htdocs\app_web\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno))
#10 C:\xampp\htdocs\app_web\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\app_web\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\app_web\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2023-10-20 17:15:48 --> Unknown column 'idGrado' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `Alumno` (`nombre`, `sexo`, `fechaNacimiento`, `idGrado`, `idGrupo`) VALUES (:nombre:, :sexo:, :fechaNacimiento:, :idGrado:, :idGrupo:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], false)
 5 APPPATH\Controllers\Alumno.php(74): CodeIgniter\Model->insert([...], false)
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Alumno->insert()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\app_web\\public\\index.php')
CRITICAL - 2023-10-20 17:19:02 --> Invalid file: "home/home.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('home/home.php')
 2 SYSTEMPATH\Common.php(1188): CodeIgniter\View\View->render('home/home', [], true)
 3 APPPATH\Controllers\Home.php(9): view('home/home')
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\app_web\\public\\index.php')
CRITICAL - 2023-10-20 17:20:24 --> Invalid file: "home/home.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('home/home.php')
 2 SYSTEMPATH\Common.php(1188): CodeIgniter\View\View->render('home/home', [], true)
 3 APPPATH\Controllers\Home.php(9): view('home/home')
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\app_web\\public\\index.php')
CRITICAL - 2023-10-20 17:22:33 --> Invalid file: "home/home.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('home/home.php')
 2 SYSTEMPATH\Common.php(1188): CodeIgniter\View\View->render('home/home', [], true)
 3 APPPATH\Controllers\Home.php(9): view('home/home')
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\app_web\\public\\index.php')
ERROR - 2023-10-20 17:29:19 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\app_web\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\app_web\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'estudiantec', 3306, '', 0)
#1 C:\xampp\htdocs\app_web\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\app_web\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\app_web\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\app_web\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\app_web\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\app_web\app\Controllers\Alumno.php(32): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\app_web\system\CodeIgniter.php(919): App\Controllers\Alumno->agregar()
#8 C:\xampp\htdocs\app_web\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno))
#9 C:\xampp\htdocs\app_web\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\app_web\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\app_web\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\app_web\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\app_web\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\app_web\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\app_web\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\app_web\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\app_web\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\app_web\app\Controllers\Alumno.php(32): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\app_web\system\CodeIgniter.php(919): App\Controllers\Alumno->agregar()
#7 C:\xampp\htdocs\app_web\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno))
#8 C:\xampp\htdocs\app_web\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\app_web\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\app_web\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-10-20 17:29:19 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `grados`', [], false)
 3 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Alumno.php(32): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Alumno->agregar()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\app_web\\public\\index.php')
ERROR - 2023-10-20 17:29:20 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\app_web\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\app_web\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'estudiantec', 3306, '', 0)
#1 C:\xampp\htdocs\app_web\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\app_web\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\app_web\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\app_web\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\app_web\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\app_web\app\Controllers\Alumno.php(32): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\app_web\system\CodeIgniter.php(919): App\Controllers\Alumno->agregar()
#8 C:\xampp\htdocs\app_web\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno))
#9 C:\xampp\htdocs\app_web\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\app_web\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\app_web\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\app_web\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\app_web\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\app_web\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\app_web\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\app_web\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\app_web\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\app_web\app\Controllers\Alumno.php(32): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\app_web\system\CodeIgniter.php(919): App\Controllers\Alumno->agregar()
#7 C:\xampp\htdocs\app_web\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno))
#8 C:\xampp\htdocs\app_web\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\app_web\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\app_web\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-10-20 17:29:20 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `grados`', [], false)
 3 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Alumno.php(32): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Alumno->agregar()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\app_web\\public\\index.php')
ERROR - 2023-10-20 17:31:42 --> mysqli_sql_exception: Unknown column 'idGrado' in 'field list' in C:\xampp\htdocs\app_web\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\app_web\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `Al...', 0)
#1 C:\xampp\htdocs\app_web\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `Al...')
#2 C:\xampp\htdocs\app_web\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `Al...')
#3 C:\xampp\htdocs\app_web\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `Al...', Array, false)
#4 C:\xampp\htdocs\app_web\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\app_web\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\app_web\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\app_web\app\Controllers\Alumno.php(74): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\app_web\system\CodeIgniter.php(919): App\Controllers\Alumno->insert()
#9 C:\xampp\htdocs\app_web\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno))
#10 C:\xampp\htdocs\app_web\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\app_web\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\app_web\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2023-10-20 17:31:42 --> Unknown column 'idGrado' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `Alumno` (`nombre`, `sexo`, `fechaNacimiento`, `idGrado`, `idGrupo`) VALUES (:nombre:, :sexo:, :fechaNacimiento:, :idGrado:, :idGrupo:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], false)
 5 APPPATH\Controllers\Alumno.php(74): CodeIgniter\Model->insert([...], false)
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Alumno->insert()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\app_web\\public\\index.php')
ERROR - 2023-10-20 17:35:10 --> mysqli_sql_exception: Unknown column 'idGrado' in 'field list' in C:\xampp\htdocs\app_web\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\app_web\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `Al...', 0)
#1 C:\xampp\htdocs\app_web\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `Al...')
#2 C:\xampp\htdocs\app_web\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `Al...')
#3 C:\xampp\htdocs\app_web\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `Al...', Array, false)
#4 C:\xampp\htdocs\app_web\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\app_web\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\app_web\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\app_web\app\Controllers\Alumno.php(74): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\app_web\system\CodeIgniter.php(919): App\Controllers\Alumno->insert()
#9 C:\xampp\htdocs\app_web\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno))
#10 C:\xampp\htdocs\app_web\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\app_web\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\app_web\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2023-10-20 17:35:10 --> Unknown column 'idGrado' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `Alumno` (`nombre`, `sexo`, `fechaNacimiento`, `idGrado`, `idGrupo`) VALUES (:nombre:, :sexo:, :fechaNacimiento:, :idGrado:, :idGrupo:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], false)
 5 APPPATH\Controllers\Alumno.php(74): CodeIgniter\Model->insert([...], false)
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Alumno->insert()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\app_web\\public\\index.php')
ERROR - 2023-10-20 17:35:33 --> mysqli_sql_exception: Unknown column 'idGrado' in 'field list' in C:\xampp\htdocs\app_web\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\app_web\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `Al...', 0)
#1 C:\xampp\htdocs\app_web\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `Al...')
#2 C:\xampp\htdocs\app_web\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `Al...')
#3 C:\xampp\htdocs\app_web\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `Al...', Array, false)
#4 C:\xampp\htdocs\app_web\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\app_web\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\app_web\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\app_web\app\Controllers\Alumno.php(74): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\app_web\system\CodeIgniter.php(919): App\Controllers\Alumno->insert()
#9 C:\xampp\htdocs\app_web\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno))
#10 C:\xampp\htdocs\app_web\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\app_web\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\app_web\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2023-10-20 17:35:33 --> Unknown column 'idGrado' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `Alumno` (`nombre`, `sexo`, `fechaNacimiento`, `idGrado`, `idGrupo`) VALUES (:nombre:, :sexo:, :fechaNacimiento:, :idGrado:, :idGrupo:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], false)
 5 APPPATH\Controllers\Alumno.php(74): CodeIgniter\Model->insert([...], false)
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Alumno->insert()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\app_web\\public\\index.php')
CRITICAL - 2023-10-20 18:36:45 --> Invalid file: "home/home.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('home/home.php')
 2 SYSTEMPATH\Common.php(1188): CodeIgniter\View\View->render('home/home', [], true)
 3 APPPATH\Controllers\Home.php(9): view('home/home')
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\app_web\\public\\index.php')
CRITICAL - 2023-10-20 18:41:12 --> Invalid file: "home/home.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('home/home.php')
 2 SYSTEMPATH\Common.php(1188): CodeIgniter\View\View->render('home/home', [], true)
 3 APPPATH\Controllers\Home.php(9): view('home/home')
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\app_web\\public\\index.php')
